contracts/utils/ItsHubAddressTracker.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import { IItsHubAddressTracker } from '../interfaces/IItsHubAddressTracker.sol';
+
+abstract contract ItsHubAddressTracker is IItsHubAddressTracker {
```

Comment:
add a docstring and explaining the motivation behind this, what the exact gas tradeoff is